config/services.yaml

```diff
@@ -14,10 +14,6 @@ services:
     _defaults:
         autowire: true      # Automatically injects dependencies in your services.
         autoconfigure: true # Automatically registers your services as commands, event subscribers, etc.
-        bind:               # defines the scalar arguments once and apply them to any service defined/created in this file
-            string $locales: '%app_locales%'
-            string $defaultLocale: '%locale%'
-            string $emailSender: '%app.notifications.email_sender%'
```

Comment:
I would do it only for this one as ultra specific param
other are more generic

Reply:
Ok, I reverted the 2 first.